cmd/apps/skychat/chat.go

```diff
@@ -56,7 +59,7 @@ func main() {
 	conns = make(map[cipher.PubKey]net.Conn)
 	go listenLoop()
 
-	http.Handle("/", http.FileServer(FS(false)))
+	http.Handle("/", http.FileServer(getFileSystem(false)))
```

Comment:
What are we using the live mode for? Not even sure if we need it. @Senyoret1 did you ever use that?

Reply:
The previous version had the option so I kept it for the new one.